### ISSUE TITLE
NTR: submandibular region

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/uberon#1137]
NTR: submandibular region
Parent: Anatomical space (Mark- do you think this is the appropriate parent?)
Definition: The region between the mandible and the hyoid bone contains the submandibular and sublingual glands, suprahyoid muscles, submandibular ganglion, and lingual artery.
syn: submandibular space
Ref: https://www.dartmouth.edu/~humananatomy/part_8/chapter_49.html

needed for HPO term: HP_0410013 Abnormality of the submandibular region

**Comment by cmungall** [comment]
I don't think you want a space; this is a subdivision of the face I think

@RDruzinsky look OK from a comparative POV?

**Comment by cmungall** [comment]
This would potentially be useful for axiomatizing

 * [UBERON:0001736](http://purl.obolibrary.org/obo/UBERON_0001736) ! submandibular gland
 * [UBERON:0002059](http://purl.obolibrary.org/obo/UBERON_0002059) ! submandibular ganglion
 * [UBERON:0002520](http://purl.obolibrary.org/obo/UBERON_0002520) ! submandibular lymph node

or would that be too simplistic? Does the gland or node ever migrate phylogenetically but retain the same name?

**Comment by RDruzinsky** [comment]
I agree, Chris. I was going to ask whether region and space are synonymous.
I don't think that they are.
Yes, I think that it is a subdivision of the neck. E.g. submandibular
triangle

Robert E. Druzinsky, Ph.D.
Clinical Associate Professor
Dept. of Oral Biology
College of Dentistry
University of Illinois at Chicago
801 S. Paulina
Chicago, IL 60612
druzinsk@uic.edu

Office: 312-996-0406
Lab: 312-996-0629
Website: www.peerj.com/RobertDruzinsky

**Comment by cewall** [comment]
I agree that it is in the neck and is submandibular triangle. Not sure about whether the salivary gland can occupy different positions in different species...

Chris Wall

**Comment by RDruzinsky** [comment]
Interesting question, @CWall

Robert E. Druzinsky, Ph.D.
Clinical Associate Professor
Dept. of Oral Biology
College of Dentistry
University of Illinois at Chicago
801 S. Paulina
Chicago, IL 60612
druzinsk@uic.edu

Office: 312-996-0406
Lab: 312-996-0629
Website: www.peerj.com/RobertDruzinsky

**Comment by cmungall** [comment]
![1178px-musculi_coli_base _my_edits_for_tringles _labeled_triangles _submandib svg](https://cloud.githubusercontent.com/assets/50745/9214158/084b87ea-404f-11e5-9475-cbd0b526e88a.png)

"Musculi coli base, my edits for tringles, labeled triangles, Submandib" by Olek Remesz (wiki-pl: Orem, commons: Orem) Modified by user:madhero88 - original image File:Musculi coli base.svg. Licensed under CC BY 3.0 via Wikimedia Commons - https://commons.wikimedia.org/wiki/File:Musculi_coli_base,_my_edits_for_tringles,_labeled_triangles,_Submandib.svg#/media/File:Musculi_coli_base,_my_edits_for_tringles,_labeled_triangles,_Submandib.svg

**Comment by cmungall** [comment]
human specific?
https://en.wikipedia.org/wiki/Submandibular_triangle#Boundaries_and_coverings

The hyoid/mandible boundaries are clear for external portions; internally sticking with the Mylohyoideus anteriorly, and by the hyoglossus posteriorly makes sense across mammals?

**Comment by RDruzinsky** [comment]
Probably true, but Chris W. and I are just not sure if the contents of the
triangle are the same across mammals.

Robert E. Druzinsky, Ph.D.
Clinical Associate Professor
Dept. of Oral Biology
College of Dentistry
University of Illinois at Chicago
801 S. Paulina
Chicago, IL 60612
druzinsk@uic.edu

Office: 312-996-0406
Lab: 312-996-0629
Website: www.peerj.com/RobertDruzinsky

**Comment by markengelstad** [comment]
surgically, in humans, the submandibular triangle (an anatomic construct, bound by specific structures) submandibular region (part of the neck, less distinct than the triangle and broader in content), and submandibular space (potential space lined w/ cervical fascia and communicating with other fascial spaces of the head and neck) are all distinct concepts.

submandibular classes are neck classes-- the inferior border of the mandible is the transition line from face to neck.

the submandibular triangle (anatomic concept) and submandibular space (a fascial space) seem both subclasses of the submandibular region, itself a subclass of the neck.

i was uncertain how specific to get ... since this all only applies to humans (as far as i know) .

**Comment by markengelstad** [comment]
for a functionally useful classification (in humans especially for cancer/ malignant disease), we would axiomatize to the very commonly utilized "Neck Dissection classification". 
In which case, the submandibular region is known as "level IB" 

![the 6 levels of the neck with sublevels](https://cloud.githubusercontent.com/assets/6233351/9233178/8fa61a92-40e6-11e5-9a82-8a4ddb9ffab7.jpg)

**Comment by markengelstad** [comment]
in fact, if were were going to "ontologize" the widely used neck dissection classification (a group of anatomical classes used in most head/ neck cancer surgeries/ diagnostics/ staging/ imaging), what would be the ontology to do it in?

**Comment by cmungall** [comment]
The neck dissection classification feels like it's getting away from uberon. FMA may be better for human-specific surgical divisions (but we could easily place them in uberon and clearly mark as human).

This is analogous to brain atlases. I think ultimately the correct representation here is an instance-level representation of a canonical organism, using the appropriate coordinate-oriented formalism. There are probably better tools than Protege for doing this, and any ontologization of the map should be an automatic transform.

We should discuss what the appropriate level of specificity for HPO is on the HPO tracker

**Comment by cmungall** [comment]
So I may have conflated triangle and region. I'm not sure we really need both classes in uberon. I would propose Uberon keeps region, and keeps the FMA class as a subclass, even though it's not strictly subclass.

**Comment by markengelstad** [comment]
i agree with that: 

So I may have conflated triangle and region. I'm not sure we really need both classes in uberon. I would propose Uberon keeps region, and keeps the FMA class as a subclass, even though it's not strictly subclass.

**Comment by cmungall** [comment]
Answering my own question about glands:

```
Domestic animals have a “mandibular gland” (or “mandibular salivary gland”) and a “mandibular lymph node”. The term "submandibular," as used in humans, is incorrect due the difference in topography of these structures.  Reference(s):  Anonymous. 
Nomina anatomica veterinaria. 4th ed. Zurich and Ithaca: World Association of Veterinary Anatomists, 1994. 
Evans HE. 
Miller's anatomy of the dog. 3rd ed. Philadelphia: WB Saunders Co, 1993
```

Source: http://www.avdc.org/Nomenclature.pdf
